e2e/integration/wordpress-01/__snapshots__/yoast-package.spec.js.snap

```diff
@@ -122,7 +122,8 @@ exports[`Yoast Package > Post > should render the schema tag #0`] = `
           "inLanguage": "en-US",
           "url": "http://2.gravatar.com/avatar/55502f40dc8b7c769880b10874abc9d0?s=96&d=mm&r=g",
           "caption": "admin"
-        }
+        },
+        "sameAs": ["http://localhost:8080"]
```

Comment:
This seems to be part of the Yoast schema, e.g.: https://yoast.com/yoast-seo-11-0/ but I don't know why this snapshot has not failed previously and only failed now when run locally.